ramips: cudy wr1300v1 reduce SPI freq to 10000000

zekica opened the pull request:
Reducing SPI flash frequency allows the build to boot on both old variants with W25Q128 chip and new variants with XM25QH128C chip.

The old 80000000 value only boots on devices with the W25Q128 flash.

This is also the change Cudy themselves made in their openwrt builds and their .dts file.

Signed-off-by: Filip Milivojevic <zekica@gmail.com>

najdanovicivan reviewed (approved):
I can confirm that this solves the issues with rootfs mounting

hauke commented on target/linux/ramips/dts/mt7621_cudy_wr1300-v1.dts:
```diff
@@ -65,7 +65,7 @@
 		reg = <0>;
-		spi-max-frequency = <80000000>;
+		spi-max-frequency = <10000000>;
 		m25p,fast-read;
```
Please remove `m25p,fast-read;` too it is only needed for ~50MHz and higher.

zekica commented:
@hauke I have removed the `m25p,fast-read` flag. I tested this on both types and it boots and works fine.

hauke commented:
Please squash this into one commit

zekica committed:
ramips: cudy wr1300v1 reduce SPI freq to 10000000

Reducing SPI flash frequency allows the build to boot on both old variants
with W25Q128 chip and new variants with XM25QH128C chip.

The old 80000000 value only boots on devices with the W25Q128 flash.

This is also the change Cudy themselves made in their openwrt builds and
their .dts file.

Removed m25p,fast-read as it is not needed with slower speeds.

Signed-off-by: Filip Milivojevic <zekica@gmail.com>
